### ISSUE TITLE
[Bug]: Crate unabble to be carried

### ISSUE THREAD

**Issue by GameDevKitY** [official-antistasi-community/A3-Antistasi#3165]
### Describe the bug

Randomly you wont see the carry option in the scrollbar for the loot crate

### How to reproduce

Unknown due to the fact its random but it persists via save loads

### Version

Aug 27, 2023 @ 12:17pm 

### Have you altered the code?

No

### Map

Atlis

### What server?

LAN Hosted

### Time bug occured (Server time/UTC)

Constant

### Mods

```html
3CB Factions 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1673456286
A3 Thermal Improvement 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2041057379
ace 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=463939057
Advanced Towing 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=639837898
Advanced Vault System: Remastered 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2794721649
-AMZ- Improved WS Sounds V7.7 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2992224248
Animated Recoil coefficient changer 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2623341670
Antistasi Plus 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2912941775
Arma 3 Performance Extension 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2565965887
Better Inventory 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2791403093
Brush Clearing 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1889104923
CBA_A3 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=450814997
CH View Distance 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=837729515
Enhanced Movement 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=333310405
ETS - Enemy Tagging System 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2057294714
Gruppe Adler Trenches 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1224892496
LAMBS_Danger.fsm 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1858075458
LAMBS_RPG 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1858070328
LAMBS_RPG_RHS 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2521192995
LAMBS_Suppression 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1808238502
LAMBS_Turrets 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1862208264
Prone Launcher 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1841047025
Real Engine 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=2127693591
Remove stamina 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=632435682
Remove stamina - ACE 3 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=782415569
RHSAFRF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843425103
RHSGREF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843593391
RHSSAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843632231
RHSUSAF 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=843577117
VET_Unflipping 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=1703187116
WMO - Walkable Moving Objects 	Steam 	https://steamcommunity.com/sharedfiles/filedetails/?id=925018569
Zeus Enhanced
```

**Comment by jaj22** [comment]
That's Advanced Vault System. To prevent carrying multiple objects at once, Antistasi will block carrying if you have any objects attached to you. Advanced Vault System apparently attaches a helper object to units.

We should transition to a blacklist system where the two different Antistasi carrying systems (HQ objects, utility items), unconscious dragging, tow ropes and various ACE carrying methods are accounted for specifically rather than a general attached-objects check. It's a lot of different systems to account for though.

**Comment by jaj22** [comment]
Fixed in #3210
